doc/thirdpartypackages/index.rst

```diff
@@ -5,75 +5,105 @@
 
-.. _toolkit_mpldatacursor:
+.. _toolkit_seaborn:
 
-MplDataCursor
-=============
+seaborn
+=======
 
-`MplDataCursor <https://github.com/joferkington/mpldatacursor>`_ is a
-toolkit written by Joe Kington to provide interactive "data cursors"
-(clickable annotation boxes) for matplotlib.
+`seaborn <http://web.stanford.edu/~mwaskom/software/seaborn>`_ is a high
+level interface for drawing statistical graphics with matplotlib. It
+aims to make visualization a central part of exploring and
+understanding complex datasets.
 
+.. image:: /_static/seaborn.png
+    :height: 157px
 
-.. _toolkit_natgrid:
 
-Natgrid
-=======
+.. _toolkit_holoviews:
 
-mpl_toolkits.natgrid is an interface to natgrid C library for gridding
-irregularly spaced data.  This requires a separate installation of the
-`natgrid toolkit <http://github.com/matplotlib/natgrid>`__.
+holoviews
+=========
 
+`holoviews <http://holoviews.org>`_ makes it easier to visualize data
+interactively, especially in a `Jupyter notebook
+<http://jupyter.org>`_, by providing a set of declarative
+plotting objects that store your data and associated metadata.  Your
+data is then immediately visualizable alongside or overlaid with other
+data, either statically or with automatically provided widgets for
+parameter exploration.
 
-.. _toolkit_matplotlibvenn:
+.. image:: /_static/holoviews.png
+    :height: 354px
 
-Matplotlib-Venn
-===============
 
-`Matplotlib-Venn <https://github.com/konstantint/matplotlib-venn>`_ provides a set of functions for plotting 2- and 3-set area-weighted (or unweighted) Venn diagrams.
+.. _toolkit_ggplot:
 
-mplstereonet
-===============
+ggplot
+======
 
-`mplstereonet <https://github.com/joferkington/mplstereonet>`_ provides stereonets for plotting and analyzing orientation data in Matplotlib.
+`ggplot <http://ggplot.yhathq.com>`_ is a port of the R ggplot2
+to python based on matplotlib.
 
+.. image:: /_static/ggplot.png
+    :height: 195px
 
-.. _hl_plotting:
+.. _toolkits-mapping:
 
-High-Level Plotting
-*******************
+Mapping Toolkits
+****************
 
-Several projects have started to provide a higher-level interface to
-matplotlib.  These are independent projects.
+Two independent mapping toolkits are available.
 
-.. _toolkit_seaborn:
+.. _toolkit_basemap:
 
-seaborn
+Basemap
 =======
 
-`seaborn <http://web.stanford.edu/~mwaskom/software/seaborn>`_ is a high
-level interface for drawing statistical graphics with matplotlib. It
-aims to make visualization a central part of exploring and
-understanding complex datasets.
+Plots data on map projections, with continental and political
+boundaries, see `basemap <http://matplotlib.org/basemap>`_
+docs.
 
-.. _toolkit_ggplot:
+.. image:: /_static/basemap_contour1.png
+    :height: 400px
 
-ggplot
-======
 
-`ggplot <https://github.com/yhat/ggplot>`_ is a port of the R ggplot2
-to python based on matplotlib.
+
+Cartopy
+=======
+
+An alternative mapping library written for matplotlib ``v1.2`` and beyond.
```

Comment:
While you're here, can you change this to 1.3? The next release will drop support for 1.2.